sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keyvault_keys.hpp

```diff
@@ -4,14 +4,11 @@
 /**
  * @brief Includes all public headers from Azure Key Vault Keys SDK library.
  *
+ * @remark This header file does exist for compatibility with prior stable releases.
+ * It is recommended to use `keys.hpp` instead.
+ *
  */
 
 #pragma once
 
-#include "azure/keyvault/keys/cryptography/cryptography_client.hpp"
-#include "azure/keyvault/keys/cryptography/cryptography_client_models.hpp"
-#include "azure/keyvault/keys/cryptography/cryptography_client_options.hpp"
-#include "azure/keyvault/keys/dll_import_export.hpp"
-#include "azure/keyvault/keys/key_client.hpp"
-#include "azure/keyvault/keys/key_client_models.hpp"
-#include "azure/keyvault/keys/key_client_options.hpp"
+#include "azure/keyvault/keys.hpp"
```

Comment:
If we really want to do this going forward, let's come up with a naming guideline to enforce this for future SDKs. We may want to leave already shipping SDKs like Keyvault Keys, as is. It's a small wart, imo, and not worth introducing multiple "top-level" header files that would require us to update samples. We now end up with multiple, which defeats the purpose and not worth the trade-off.

Reply:
We do update all the samples to use new. The old one is quietly there, because we can't break it. I think it is the right way.

The rule should be, if the headers are in `azure/service/...` directory, the all-inclusive header is `azure/service.hpp`